api/src/services/payload.ts

```diff
@@ -208,20 +208,25 @@ export class PayloadService {
 	 */
 	processGeometries<T extends Partial<Record<string, any>>[]>(payloads: T, action: Action): T {
 		const helper = getGeometryHelper();
+
 		const process =
 			action == 'read'
-				? (value: any) => wktToGeoJSON(value)
+				? (value: any) => {
+						if (typeof value === 'string') return wktToGeoJSON(value);
```

Comment:
Is that check necessary? `value`  should always be a string (or undefined), as it is the result of `geometryHelper.asText(column)`.

Reply:
There's a case where the payload is processing the value as a read coming from the user's payload on create/update (to prepare the `delta` values). In that case, the value is already a GeoJSON object, which crashes the wktToGeoJSON function.